dev/PushNotifications/PushNotificationsLongRunningTask/NotificationListenerManager.cpp

```diff
@@ -0,0 +1,44 @@
+﻿#include "pch.h"
+
+using namespace Microsoft::WRL;
+
+void NotificationListenerManager::Initialize(std::shared_ptr<ForegroundSinkManager> foregroundSinkManager, std::vector<std::wstring>& processNameList)
+{
+    m_foregroundSinkManager = foregroundSinkManager;
+
+    for (std::wstring& processName : processNameList)
+    {
+        AddListener(processName);
+    }
+}
+
+void NotificationListenerManager::AddListener(std::wstring processName)
```

Comment:
AddAppActivator: If we decide to go with AppActivator instead of NotificationListener